helix-term/src/commands.rs

```diff
@@ -1019,6 +1019,30 @@ mod cmd {
         doc.format(view.id)
     }
 
+    fn earlier(editor: &mut Editor, args: &[&str], event: PromptEvent) {
+        let sotp = match args.join(" ").parse::<helix_core::history::UndoKind>() {
```

Comment:
Can you please reword `sotp` to `kind` or something that is more descriptive? I think you forgot to change this.

Reply:
Changed to uk for UndoKind. This is consistent with how it's called elsewhere.